src/components/beat.js

```diff
@@ -1,13 +1,13 @@
-import {PUNCH_OFFSET, SWORD_OFFSET} from './beat-generator';
+import { PUNCH_OFFSET, SWORD_OFFSET } from './beat-generator';
```

Comment:
spaces around brackets should be gone. maybe this is a different case because it's not an object?